lib/request-validation.ts

```
@@ -0,0 +1,33 @@
+    headers?: Record<string, string>;
+    body?: Record<string, unknown>;
+    maxSize?: number | undefined;
+    allowedContentTypes?: string[];
+  },
+) {
+  const contentLength = request.headers.get("content-length");
+  const contentType = request.headers.get("content-type");
+
+  if (
+    contentLength &&
+    parseInt(contentLength) > (options.maxSize || 1024 * 1024)
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)